common/app/views/support/Profile.scala

```diff
@@ -129,6 +129,7 @@ object FacebookOpenGraphImage extends ShareImage(FacebookShareImageLogoOverlay.i
 
 object EmailImage extends Profile(width = Some(580), autoFormat = false) {
   override val qualityparam = "q=40"
+  val knownWidth = 580
```

Comment:
I suppose you could do `knownWidth = width.get`?

I don't think doing `EmailImage.get` is too bad either since you know which subtype is used in the Helpers.  If later somebody tries to do the same thing with another subtype with `None` for width, it will still compile - but should be pretty obviously broken at runtime.